docusaurus.config.js

```diff
@@ -27,19 +28,53 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          path: 'content',
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl:
             'https://github.com/kubeshop/botkube-docs/edit/main/',
+          versions: {
+            current: {
+              label: `Unreleased 🚧`,
+            },
+          },
         },
         theme: {
-          customCss: require.resolve('./src/css/custom.css'),
+          customCss: require.resolve('./src/css/custom.scss'),
         },
       }),
     ],
   ],
 
-  plugins: ['docusaurus-plugin-sass'],
+  scripts: [
+    {
+      src:
+        'https://kit.fontawesome.com/00670398ef.js',
+      async: false,
+    },
+  ],
```

Comment:
We can think about using https://remixicon.com/ or similar, and copy+paste SVGs directly to the page 🙂

Reply:
Paweł can we postpone it until the full rebranding will be done?

Reply:
👍 Sure thing